test/gtest/core/test_mc_reduce.cc

```diff
@@ -57,6 +57,9 @@ class test_mc_reduce : public testing::Test {
             }
 #endif
             break;
+        case UCC_MEMORY_TYPE_CUDA_MANAGED:
```

Comment:
why do we treat Cuda managed differently than Cuda here ? What with the triggered case ?
If there is not difference then you can simply add `case UCC_MEMORY_TYPE_CUDA_MANAGED:` below `case UCC_MEMORY_TYPE_CUDA:`

Reply:
we don't support triggered with cudaManaged